R/missingDataImputation.R

```diff
@@ -235,15 +231,19 @@ MissingDataImputation <- function(jaspResults, dataset, options) {
 ###------------------------------------------------------------------------------------------------------------------###
 
 .createTracePlot <- function(convergencePlots, miceMids) {
-  # if (!is.null(jaspResults[["tracePlot"]])) return()
+
+  #if (!is.null(jaspResults[["tracePlot"]])) return()
 
   tracePlot <- createJaspPlot(title = "Trace Plot", height = 320, width = 480)
-  # tracePlot$dependOn(options = "variables")
-  
-  # Bind plot to jaspResults via the convergencePlots container:
+  tracePlot$dependOn(options = "variables")
```

Comment:
We don't want to depend on `options$variables` since that field doesn't exist. Our equivalent would be `options$imputationVariables`, but that isn't the right dependency either because you wouldn't get updates when changing mice options, only when changing the imputation targets. 

We don't need to solve this problem in this PR, though. This is an issue to address when fixing the overall object updating problems.

Reply:
Solved in [85854c6](https://github.com/jasp-missing-data/jaspMissingData/pull/18/commits/85854c6497f3fba4d8cb23ef7db422ff54da2882)